AsyncDisplayKit/Details/ASCollectionItem.mm

```diff
@@ -51,10 +49,9 @@ - (ASCellNode *)node
     ASCellNode *node = _nodeBlock();
     _nodeBlock = nil;
     if (node == nil) {
-      ASDisplayNodeFailAssert(@"Node block returned nil node! Index path: %@", _indexPath);
+      ASDisplayNodeFailAssert(@"Node block returned nil node!");
       node = [[ASCellNode alloc] init];
     }
-    node.cachedIndexPath = _indexPath;
```

Comment:
The loss of this encache is notable, but it will probably have no runtime impact. This was always a hack, and we'll find a better way to quickly look up the index path for a cell node (probably using a `ASCellNode -> NSIndexPath` map table in the data controller).